wiki/Contests/Mapping_Olympiad/en.md

```diff
@@ -0,0 +1,69 @@
+
+## Prizes
+
+The osu! Mapping Olympiad contest has its own rewarding system and work under a point system, as the Monthly Beatmapping Contest does.
+
+* **Contest winner**: 6 points
+* **Second place**: 4 points
+* **Third place**: 2 points
+
+Reaching points can grant the player specific prizes:
+
+* As of the second contest's edition, winning a contest grants you an *Elite Mapper* profile title.  
+* Once a player earns 6 points, they get rewarded with a **profile badge**, that can progress up to 5 times on every 6 points the player gets.
+* Every top 3 contestants from each contest earns a **supporter tag** of **two**, **three** or **four months** depending of a player ranking. 
+## Points
+
+An other leaderboard with more details about each contestant's track record below is available [here](https://docs.google.com/spreadsheets/d/1_gIDJwTOgIhGr2h4069-r1C-2GHuV--5wtwPywzYsz8/edit?usp=sharing).
+
+| User(s) | Points |
+| :-- | :-- |
+| ![][flag_SG] [Sinnoh](https://osu.ppy.sh/users/4236057), ![][flag_KR] [Kawawa](https://osu.ppy.sh/users/4647754) | 10 |
+| ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418), ![][flag_IT] [Bonzi](https://osu.ppy.sh/users/1313969), ![][flag_CA] [Monstrata](https://osu.ppy.sh/users/2706438), ![][flag_FR] [Nofool](https://osu.ppy.sh/users/672430), ![][flag_FI] [PianoLuigi](https://osu.ppy.sh/users/9665915), ![][flag_CN] [Necho](https://osu.ppy.sh/users/4086593), ![][flag_CN] [Ryuusei Aika](https://osu.ppy.sh/users/7777875), ![][flag_HK] [alienflybot](https://osu.ppy.sh/users/636114), ![][flag_US] [Toaph Daddy](https://osu.ppy.sh/users/7616811)  | 6 |
+| ![][flag_UA] [Fallmorph](https://osu.ppy.sh/users/5720389), ![][flag_SG] [Rokkea](https://osu.ppy.sh/users/2500099), ![][flag_CA] [Kibbleru](https://osu.ppy.sh/users/3193504), ![][flag_US] [Nifty](https://osu.ppy.sh/users/4956097), ![][flag_KR] [Minato Yukina](https://osu.ppy.sh/users/531253), ![][flag_BE] [yaspo](https://osu.ppy.sh/users/4945926) | 4 |
+| ![][flag_CA] [Gaia](https://osu.ppy.sh/users/2683648), ![][flag_CN] [fanzhen0019](https://osu.ppy.sh/users/418699), ![][flag_FR] [Nozhomi](https://osu.ppy.sh/users/2716981), ![][flag_ID] [Volta](https://osu.ppy.sh/users/4154071), ![][flag_ID] [\_VianK_](https://osu.ppy.sh/users/6889055), ![][flag_US] [Ascendance](https://osu.ppy.sh/users/2931883), ![][flag_CA] [J1_](https://osu.ppy.sh/users/5918561), ![][flag_HK] [how2miss](https://osu.ppy.sh/users/4477199), ![][flag_CN] [Yumeno Himiko](https://osu.ppy.sh/users/1806962), [lenpai](https://osu.ppy.sh/users/5314573), ![][flag_CA] [BringoBrango](https://osu.ppy.sh/users/10274043) | 2 |
```

Comment:
```suggestion
| User(s) | Points |
| :-- | :-- |
| ![][flag_SG] [Sinnoh](https://osu.ppy.sh/users/4236057), ![][flag_KR] [Kawawa](https://osu.ppy.sh/users/4647754) | 10 |
| ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418), ![][flag_IT] [Bonzi](https://osu.ppy.sh/users/1313969), ![][flag_CA] [Monstrata](https://osu.ppy.sh/users/2706438), ![][flag_FR] [Nofool](https://osu.ppy.sh/users/672430), ![][flag_FI] [PianoLuigi](https://osu.ppy.sh/users/9665915), ![][flag_CN] [Necho](https://osu.ppy.sh/users/4086593), ![][flag_CN] [Ryuusei Aika](https://osu.ppy.sh/users/7777875), ![][flag_HK] [alienflybot](https://osu.ppy.sh/users/636114), ![][flag_US] [Toaph Daddy](https://osu.ppy.sh/users/7616811) | 6 |
| ![][flag_UA] [Fallmorph](https://osu.ppy.sh/users/5720389), ![][flag_SG] [Rokkea](https://osu.ppy.sh/users/2500099), ![][flag_CA] [Kibbleru](https://osu.ppy.sh/users/3193504), ![][flag_US] [Nifty](https://osu.ppy.sh/users/4956097), ![][flag_KR] [Minato Yukina](https://osu.ppy.sh/users/531253), ![][flag_BE] [yaspo](https://osu.ppy.sh/users/4945926) | 4 |
| ![][flag_CA] [Gaia](https://osu.ppy.sh/users/2683648), ![][flag_CN] [fanzhen0019](https://osu.ppy.sh/users/418699), ![][flag_FR] [Nozhomi](https://osu.ppy.sh/users/2716981), ![][flag_ID] [Volta](https://osu.ppy.sh/users/4154071), ![][flag_ID] [\_VianK\_](https://osu.ppy.sh/users/6889055), ![][flag_US] [Ascendance](https://osu.ppy.sh/users/2931883), ![][flag_CA] [J1\_](https://osu.ppy.sh/users/5918561), ![][flag_HK] [how2miss](https://osu.ppy.sh/users/4477199), ![][flag_CN] [Yumeno Himiko](https://osu.ppy.sh/users/1806962), [lenpai](https://osu.ppy.sh/users/5314573), ![][flag_CA] [BringoBrango](https://osu.ppy.sh/users/10274043) | 2 |
```
the listing looks too cluttered though. would actually prefer something similar to the one on the spreadsheet - contestant by contestant

Reply:
I had it like this cause the list was *a bit* too long and it kinda looked weirdo